docs/PUBLISHING.md

```diff
@@ -37,14 +23,9 @@ Update Channels
 ---------------
 
 Etcher has a setting to include the unstable update channel. If this option is
```

Comment:
Presumably in a later PR, this will change to `Etcher has a setting to include the prerelease update channel.` ? (with the name of the actual setting being changed too)